### PR TITLE
fix guidebook for Trade Station

### PR DESCRIPTION
**Pull request by Hmeister-real**
Did the <Document> thing wrong

## About the PR
 Did an oopsy and had an extra space that shouldnt have been there

## Why / Balance
It didnt work, now it should

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

 no cl no fun

### CODE REVIEW

**Comment by Hmeister-real** [comment]
Extremely minor fix to my og pr, literally one change. Please merg

**Comment by EmoGarbage404** [comment]
did you test that this works? what was even broken?

**Comment by Hmeister-real** [comment]
> did you test that this works? what was even broken?

the only problem was that TradeStation.xml wasnt recognised as a document so wasnt listed in the guidebook because of an extra space.

File: Resources/ServerInfo/Guidebook/Cargo/TradeStation.xml [hunk 1]
```diff
@@ -7,5 +7,4 @@
   
   ## Collecting Orders
   All cargo orders are sent to the [color=#a4885c]Trade Station[/color] to be [color=#a4885c]collected[/color], this includes items sent by centcom. To collect them you have to travel to the Trade station and they should be there on the cargo pallets in a crate, ready to be taken back to the station.
-
-  <Document>
+<Document>
```
Comment (Partmedia) [inline Resources/ServerInfo/Guidebook/Cargo/TradeStation.xml:10]: Shouldn't this be a `</Document>`?
```suggestion
</Document>
```

**Review by Partmedia — CHANGES_REQUESTED** [review]

**Review by Partmedia — APPROVED** [review]

**Pull request merged**